site/src/pages/RoadmapPage/sidebar/GERequiredCourseList.tsx

```diff
@@ -4,9 +4,25 @@ import trpc from '../../../trpc';
 import RequirementsLoadingIcon from './RequirementsLoadingIcon';
 import { setGERequirements } from '../../../store/slices/courseRequirementsSlice';
 import { useAppDispatch, useAppSelector } from '../../../store/hooks';
+import { findCommonLabelPrefix } from '../../../helpers/substitutions';
 
-function getCoursesForGE() {
-  return trpc.programs.getRequiredCoursesUgrad.query({ id: 'GE' });
+async function getCoursesForGE() {
+  const fetchedCourses = await trpc.programs.getRequiredCoursesUgrad.query({ id: 'GE' });
```

Comment:
variable name is inaccurate, as this is a requirements list.

Additionally, there is already a function to normalize responses, so the logic written here might be better suited to go alongside other logic that does similar things. Even if you do only apply it to GEs, it still might be worth defining a separate function to normalize its data there